python/seldon_core/microservice.py

```diff
@@ -462,13 +481,42 @@ def grpc_prediction_server():
         except (NotImplementedError, AttributeError):
             pass
 
-        server.add_insecure_port(f"0.0.0.0:{grpc_port}")
-
+        server.add_insecure_port(bind_address)
         server.start()
+        _wait_forever(server)
+
+    @contextlib.contextmanager
+    def _reserve_grpc_port():
```

Comment:
I can see that the custom metrics test is failing, which suggests that if this is indeed the case, this may be interfering with the prometheus metrics port. Is there a way to ensure that the port ranges of this are limited to the 5000-5999 range? that way we'd be able to avoid clashing

Reply:
Hmm nevermind, just realised it is actually on the same port